plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StringVersionComparator.java

```diff
@@ -19,7 +19,7 @@
 import java.util.Comparator;
 
 /**
- * A {@link Comparator<String>} for the Faceted Project framework that compares versions that are
```

Comment:
I think Javadoc isn't smart enough to link to generics from an `{@ link}`. In any case, Eclipse flags this as a warning.